content/apps/using-oauth2.md

```diff
@@ -0,0 +1,157 @@
+# Using OAuth 2.0 to connect an App
+
+The PIM uses OAuth 2.0 to manage the authorization of Apps.
+
+![App activation diagram](../img/apps/app-activation-sequence-diagram.png)
+
+
+## Credentials
+
+To begin, you must obtain valid OAuth 2.0 client credentials by registering your App on the
+[Akeneo Marketplace](https://marketplace.akeneo.com/node/add/extension).
+
+Then, when any user want to connect his PIM to your App, the authorization process will go through the following steps.
+
+::: info
+In the following examples, we will use those fictional urls:  
```

Comment:
```suggestion
In the following examples, we will use these fictional URLs:  
```
[for the context if interested](http://www.differencebetween.net/language/difference-between-these-and-those/)